packages/lib/src/interactions/AxialSelectToZoom.tsx

```diff
@@ -15,26 +14,16 @@ interface Props extends CommonInteractionProps {
 function AxialSelectToZoom(props: Props) {
   const { axis, modifierKey, disabled } = props;
 
-  const { canvasSize, visRatio } = useVisCanvasContext();
-  const moveCameraTo = useMoveCameraTo();
+  const { visRatio } = useVisCanvasContext();
+  const zoomOnBox = useZoomOnBox();
```

Comment:
The function returned by this new hook updates the camera's scale and calls `moveCameraTo`. It removes duplication between `SelectToZoom` and `AxialSelectToZoom`.